array.c

```diff
@@ -1567,15 +1606,16 @@ ary_ensure_room_for_unshift(VALUE ary, int argc)
 
 /*
  *  call-seq:
- *     ary.unshift(obj, ...)  -> ary
- *     ary.prepend(obj, ...)  -> ary
+ *    array.unshift(*objects) -> self
+ *    array.prepend(*objects) -> self
  *
- *  Prepends objects to the front of +self+, moving other elements upwards.
- *  See also Array#shift for the opposite effect.
```

Comment:
Can we keep the `See also`? In my experience is helps while researching.

Reply:
Done, and more.